### DISCUSSION TITLE
Organizational Folders

### DISCUSSION THREAD

**Discussion by jasonmacer** [rustdesk/rustdesk#8244] [General]
Good morning all! 

I want to say thank you, thank you, thank you! for this tool! Especially the ability to host it locally on my own environment. Supporting healthcare clients is always a nightmare, and this keeps them happy for it not being hosted on someone else's cloud environment. 

I think we've all probably used TeamViewer, and more recently I've used something called DattoRMM. I've actually used it for years, originally it was AutoTask RMM, and then migrated to Datto. It used to be a great piece of tech, but after it was bought by Kaseya it went to heck in a handbasket.

One of the things that TeamViewer and AutoTask/Datto have is the ability to create containers or folders to organize connections. Personally, I have my Datto and Teamviewer organized in a tree-like hierarchy where I have "Sites" and then groups within those sites like the following:

* Site 1
  * Reception
  * Lab
  * Billing
  * Pharmacists

* Site 2
  * Reception
  * Compound
  * Lab
  * Billing
  * Pharmacists
  * Administration

I know the search feature exists, but having the ability to organize the connections would be amazingly helpful!

Thank you guys again for all you're doing!

-Jason
AKA TheStormsOfFury

**Comment by yaroz** [comment]
If you have the pro version, you can setup a separate site address book per site, and have tags for functional levels.
